packages/test-runner-intercept/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@web/test-runner-intercept",
+  "version": "0.1.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Plugin for intercepting modules in @web/test-runner",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/modernweb-dev/web.git",
+    "directory": "packages/test-runner-intercept"
+  },
+  "author": "modern-web",
+  "homepage": "https://github.com/modernweb-dev/web/tree/master/packages/test-runner-intercept",
+  "main": "browser/index.js",
+  "exports": {
+    ".": {
+      "types": "./browser/index.d.ts",
+      "default": "./browser/index.mjs"
+    },
+    "./plugin": {
```

Comment:
Package exports should use file extensions

Reply:
I tend to agree, but I reused the same convention used in `test-runner-commands`, `test-runner-core` and `test-runner-visual-regression`. Maybe this should be done as a general overhaul (eg. the PR to move to es modules?)

Reply:
Some scripts right now, are based on these types of conventions within the project.

Reply:
if you are referring to `storybook-framework-web-components`, specifically `/preset` path, then IIRC it's a convention of the storybook
so if not required by other tools, I'd say you should add an extension

scripts indeed will mess it up a little bit, because they will ad a double extension to the generated files, but I don't think that's an issue, if it is, it shouldn't be hard to fix

Reply:
No, I meant these scripts:
- [generate-mjs-dts-entrypoints.mjs](https://github.com/modernweb-dev/web/blob/master/scripts/generate-mjs-dts-entrypoints.mjs)
- [generate-ts-configs.mjs](https://github.com/modernweb-dev/web/blob/master/scripts/generate-ts-configs.mjs)

They rely heavily on projects following the same conventions. :-)

I just realized this entrypoint is a node entrypoint, not a browser entrypoint. Most node projects don't use extensions in their exports. Browser exports ofc do. Do you still want me to update it?

Reply:
yeah, I know the scripts :D I meant `reused the same convention used` was referring to that

my point is: don't look at conventions, because we are now changing them and IIRC the idea is to use extensions always, unless the extension will break something, e.g. in the `storybook-framework-web-components` `/preset` the extension broke the storybook for me

I think I tried the scripts with the exports containing extensions, they worked, and if not, you can fix a specific bug in there

Reply:
This is now updated. :-) Can you check and resolve if ok?